ui/src/components/VolumeDetailCard.js

```diff
@@ -195,6 +211,8 @@ const VolumeDetailCard = (props) => {
     <VolumeDetailCardContainer>
       <VolumeInformation>
         <VolumeNameTitle data-cy="volume_detail_card_name">
+          <CircleStatus className="fas fa-circle" status={health} />
```

Comment:
```suggestion
          <CircleStatus className="fas fa-circle fa-2x" status={health} />
```
According to the design, please use the bigger size of the icon.